rails-time-track/config/routes.rb

```diff
@@ -1,14 +1,16 @@
 Rails.application.routes.draw do
-  namespace :api do
+  namespace :api, defaults: {format: "json"} do
```

Comment:
Why do you add that code? What does it do exactly?

Reply:
I added this piece of code to set a default format.